static/app/gettingStartedDocs/dotnet/dotnet.tsx

```diff
@@ -50,10 +56,30 @@ SentrySdk.Init(options =>
 
     // This option is recommended for client applications only. It ensures all threads use the same global scope.
     // If you're writing a background service of any kind, you should remove this.
-    options.IsGlobalModeEnabled = true;
+    options.IsGlobalModeEnabled = true;${
+      params.isPerformanceSelected
+        ? `
 
     // This option will enable Sentry's tracing features. You still need to start transactions and spans.
-    options.EnableTracing = true;
+    options.EnableTracing = true;`
+        : ''
+    }${
+      params.isProfilingSelected
+        ? `
+
+    // Sample rate for profiling, applied on top of othe TracesSampleRate,
+    // e.g. 0.2 means we want to profile 20 % of the captured transactions.
+    // We recommend adjusting this value in production.
+    options.ProfilesSampleRate = 1.0;
+
+    // Requires NuGet package: Sentry.Profiling
+    // Note: By default, the profiler is initialized asynchronously. This can be tuned by passing a desired initialization timeout to the constructor.
+    options.AddIntegration(new ProfilingIntegration(
+        // During startup, wait up to 500ms to profile the app startup code. This could make launching the app a bit slower so comment it out if your prefer profiling to start asynchronously
+        TimeSpan.FromMilliseconds(500)
+    ));`
```

Comment:
docs.sentry.io references `TracesSampleRate` but these instructions use `EnableTracing`. Which one should we standardize on?

Reply:
EnableTracing is an alias to TracesSampleRate=1 but we should use the latter which is more explicit and easy to folks to adjust